docs/uma-tokenholders/approved-collateral-currencies.md

```diff
@@ -36,6 +36,18 @@ sidebar_label: Approved Collateral Currencies
 |BAL| 8 |[UMIP-72](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-72.md)
 |xSUSHI| 22 | [UMIP-67](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-67.md)
 |XIO| 1500 | [UMIP-71](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-70.md)
```

Comment:
Looks good @henrystats! Do you mind updating the collateral currency final fees that were changed in [UPP-3](https://github.com/UMAprotocol/UMIPs/blob/master/UPPs/upp-3.md)?

Reply:
yes boss! on it!

Reply:
I've updated the final fees, can you please approve so I can merge?

Reply:
Thanks!